src/value.rs

```diff
@@ -174,6 +174,15 @@ impl ValueSum {
             sign,
         )
     }
+
+    /// Commits this value with the corresponding trapdoor `rcv` by $ValueCommit^Orchard$
```

Comment:
```suggestion
    /// Commits this value with the corresponding trapdoor `rcv` by ValueCommit^Orchard.
```
(LaTeX would not render correctly here.)